media/css/quantum/_buttons.scss

```diff
@@ -20,7 +45,7 @@ a.button:visited {
         padding: .9em 40px;
 
         .download-title {
-            color: #0040ab;
+            color: #fff;
```

Comment:
I can't comment on the line below here, but the hover color feels too bright and makes the button text hard to read? I think Christa mentioned she was going to work on hover states, so she might have an alternative? I also notice there's a 2px border on hover, which may or may not be required.

Reply:
Going to go 2% darker until we have real direction.